runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -1638,5 +1639,31 @@ class TR_VectorAPIExpansion : public TR::Optimization
    *      Transformed node
    */
    static TR::Node *transformNary(TR_VectorAPIExpansion *opt, TR::TreeTop *treeTop, TR::Node *node, TR::DataType elementType, TR::VectorLength vectorLength, int32_t numLanes, handlerMode mode, TR::ILOpCodes scalarOpCode, TR::ILOpCodes vectorOpCode, int32_t firstOperand, int32_t numOperands, vapiOpCodeType opCodeType, bool transformRORtoROL);
-   };
-#endif /* VECTORAPIEXPANSION_INCL */
```

Comment:
I believe you shouldn't remove these two lines.

Reply:
Oh, sorry, some commits mix.